test/wdio.conf.js

```diff
@@ -96,30 +94,26 @@ exports.config = {
       "browserName": "IE",
       "browser_version": "11.0",
       'browserstack.sendKeys': 'true',
-      "browserstack.selenium_version": "3.5.2",
     },
     {
       "os": "Windows",
       "os_version": "7",
       "browserName": "IE",
       "browser_version": "10.0",
       'browserstack.sendKeys': 'true',
-      "browserstack.selenium_version": "3.5.2",
     },
     {
       "os": "Windows",
       "os_version": "7",
       "browserName": "IE",
       "browser_version": "9.0",
       'browserstack.sendKeys': 'true',
-      "browserstack.selenium_version": "3.5.2",
     },
     {
       "os": "Windows",
       "os_version": "7",
       "browserName": "IE",
       "browser_version": "8.0",
-      "browserstack.selenium_version": "3.5.2",
       'browserstack.sendKeys': 'true',
     },
```

Comment:
Should we try commenting these chunks out, at least temporarily, to see if everything proceeds without any hitches?

Reply:
They all passed in CI except IE 10 which I've retried since and found to work fine.

TIL I can show you that using these public links: https://automate.browserstack.com/dashboard/v2/public-build/cnplOWt2VS82TTgxdFpjQjI3YmFienBTOVVJQTBEamoxOHhEcHhWZDFmUnlzVUR4eUN4bVpvb2F4WmY4dkZrbzRpbHJ3a2dNK0xaVE5DZFViaytaQXc9PS0tVXNMU2RrY3pVZ1Q4MGpzVzFkZnkzdz09--98d72c3ba80d612b15dc4971c5cc0e13782db2bd

I think I'll try to find a way to attach that to the commit status